aten/src/TH/THTensor.hpp

```diff
@@ -8,6 +8,22 @@
 
 #include <atomic>
 
+typedef struct _THTensor
```

Comment:
This is C++, so no typedef necessary. (Same below)

Reply:
I considered that, but we haven't killed the C api yet, so it seemed better to leave this equivalent to the forward define in the C API (https://github.com/pytorch/pytorch/blob/deb56dfd06a4834434c8146100e0dcc8d91040dc/aten/src/TH/generic/THTensor.h#L10) until we kill that.  (Note _THTensor will become THTensor once we only a single THTensor type).